### PR TITLE
11.0 mig product brand

### PR DESCRIPTION
**Pull request by jprobineau**

### CODE REVIEW

File: product_brand/views/product_brand_view.xml [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+            </field>
+        </field>
+    </record>
+
+    <record id="view_product_template_kanban_brand" model="ir.ui.view">
+        <field name="name">product kanban view: add brand</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_kanban_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@class='oe_kanban_details']/strong[1]" position="after">
```
Comment (christophlsa) [inline product_brand/views/product_brand_view.xml:146]:
```
odoo.addons.base.ir.ir_ui_view: Error-prone use of @class in view product kanban view: add brand (product_brand.view_product_template_kanban_brand): use the hasclass(*classes) function to filter elements by their classes
```

File: product_brand/views/product_brand_view.xml [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_product_variant_kanban_brand" model="ir.ui.view">
+        <field name="name">product variant kanban view: add brand</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_kanban_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@class='oe_kanban_details']/strong[1]" position="after">
```
Comment (christophlsa) [inline product_brand/views/product_brand_view.xml:162]:
```
odoo.addons.base.ir.ir_ui_view: Error-prone use of @class in view product variant kanban view: add brand (product_brand.view_product_variant_kanban_brand): use the hasclass(*classes) function to filter elements by their classes
```

**Review by christophlsa — CHANGES_REQUESTED** [review]
Can you please check the two xpaths? Odoo throws two warnings. :)

**Comment by JayVora-SerpentCS** [comment]
please squash transifex commits. one of our module website_product_brand depends on this, we would need the module.

File: product_brand/README.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+* Leonardo Donelli <donelli@webmonks.it>
+* Serpent Consulting Services Pvt. Ltd. <jay.vora@serpentcs.com>
+* Marcelo Pickler <loxamir@gmail.com>
+* Andrius Laukavičius <ala@boolit.eu> (Boolit)
+* Apertoso N.V. <www.apertoso.be>
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
```
Comment (JayVora-SerpentCS) [inline product_brand/README.rst:77]: https everywhere

**Comment by Daniel-CA** [comment]
@jprobineau Have you planning to continue with the migration of this module? There is another PR with the same module (#300) and I do not want to fall into "the curse of the un-migrabable module" opening another one.

Kind regards

**Comment by JayVora-SerpentCS** [comment]
@ronnyhouben Can you squash transbot commits?

**Comment by ghost** [comment]
Hi,

 

Please elaborate… I do not get this.

 

 

Met vriendelijke groeten, Meilleures salutations, With kind regards,

 

Ronny Houben | Business Consultant

 



 

Guido Gezellelaan, 16

B-9800 - Deinze  +32 9 381 64 50

 

TIO3 - O. Delghuststraat, 60

B-9600 - Ronse   +32 55 21 80 07

 

 <tel:+32%20475%2097%2024%2056> +32 477 43 78 02

 <http://www.accomodata.be/> www.accomodata.be

**Comment by JayVora-SerpentCS** [comment]
You should merge all commits with this message "OCA Transbot updated translations from Transifex" into one.

**Comment by pedrobaeza** [comment]
See https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests for the procedure.

**Comment by pedrobaeza** [comment]
Superseeded by #340

**Pull request closed without merging**